lib/Basics/Result.h

```diff
@@ -46,6 +46,10 @@ class Result final {
    */
   Result(int errorNumber, std::string&& errorMessage) noexcept;
 
+  Result(int errorNumber, std::string_view const& errorMessage);
```

Comment:
we should also include `string_view` in this file now, which we currently don't do.